### PR TITLE
Build binaries from build_dist.sh

### PR DESCRIPTION
**Pull request by scollazo**

### CODE REVIEW

**Review by djjuhasz — CHANGES_REQUESTED** [review]
@scollazo I don't understand the purpose of this change, and the tests are not passing.  Is it still necessary?

File: hack/build_dist.sh [hunk 1]
```diff
@@ -24,17 +24,32 @@ LONG="${SHORT}$long_suffix"
...
+ "enduro-a3m-worker")
+	TARGET=enduro-a3m-worker
+        TARGETPATH=./cmd/enduro-a3m-worker/
+ ;;
+ "enduro-am-worker")
+	TARGET=enduro-am-worker
+        TARGETPATH=./cmd/enduro-am-worker/
+ ;;
+
+ esac
```
Comment (sevein) [inline hack/build_dist.sh:51]: There is a whitespace before `esac`, is that on purpose?

File: hack/build_dist.sh [hunk 2]
```diff
@@ -24,17 +24,32 @@ LONG="${SHORT}$long_suffix"
...
 VERSION_LONG="$LONG"
 VERSION_GIT_HASH="$GIT_HASH"
 VERSION_PATH="$VERSION_PATH"
 EOF
 	exit 0
-fi
+ ;;
+ "enduro")
+	TARGET=enduro
+        TARGETPATH=.
```
Comment (sevein) [inline hack/build_dist.sh:40]: Mixed use of tabs and whitespaces for indenting.

**Review by sevein — CHANGES_REQUESTED** [review]
A detailed description would be nice. I'm guessing that you want to have the following syntax:

    ./hack/build_dist.sh enduro-a3m-worker

Instead of:

    ./hack/build_dist.sh -o enduro-a3m-worker ./cmd/enduro-a3m-worker

For me the main issue with this change is that I won't be able to pass arbitrary flags to `go build`, e.g. something I've been doing recently:

    ./hack/build_dist.sh -cover -o /tmp/test-instrumented-binary ./cmd/enduro-a3m-worker

**Comment by scollazo** [comment]
> ./hack/build_dist.sh -o enduro-a3m-worker ./cmd/enduro-a3m-worker

That's all I need, closing this

**Pull request closed without merging**